NuGet.config

```diff
@@ -1,7 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetcidev/api/v3/index.json" />
+    <add key="dotnet-core" value="https://www.myget.org/F/dotnet-core/api/v3/index.json" />
```

Comment:
I'll remove this before merge. Have it here so I can see if https://github.com/dotnet/corefx/pull/3999 gets to the build in time and replace the reflection with that.